setup.py

```diff
@@ -462,7 +462,13 @@ def run(self):
         else:
             report('-- Not using cuDNN')
         if cmake_cache_vars['USE_CUDA']:
-            report('-- Detected CUDA at ' + cmake_cache_vars['CUDA_TOOLKIT_ROOT_DIR'])
+            if cmake_cache_vars['CUDA_TOOLKIT_ROOT_DIR']:
+                cuda_toolkit_root_dir = cmake_cache_vars['CUDA_TOOLKIT_ROOT_DIR']
+            elif cmake_cache_vars['CUDA_TOOLKIT_ROOT_DIR_INTERNAL']:
+                cuda_toolkit_root_dir = cmake_cache_vars['CUDA_TOOLKIT_ROOT_DIR_INTERNAL']
```

Comment:
Actually this is not necessary. It just happens that cmake_cache_vars doesn't have `CUDA_TOOLKIT_ROOT_DIR` when configured directly through cmake, but this can also be solved by giving the path explicitly on commandline. So I removed this change. Thanks!